Makefile

```diff
@@ -2,22 +2,26 @@ deps_wasm:
 	cd examples/wasm && yarn install
 	cargo install wasm-pack --version 0.8.1
 
-build_wasm:
+build_wasm: deps_wasm
```

Comment:
I removed that in the last PR. I though you saw it I left a comment.

`deps_wasm` will fail if `wasm_pack` is already install and doing `make build_wasm` won't work. It is a bit annoying when you want to build locally to test.

Reply:
sorry, I had not seen the comment. I will see if we can fix it or I will remove it again

Reply:
it is now working, you will need to update cargo on your side.